thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/tools/RunAdhocDatabaseQueriesTool.java

```diff
@@ -454,6 +455,17 @@ private void disableAllActiveFunction(Collection<Long> exception){
     }
   }
 
+  private void disableAllActiveSubsGroups(Collection<Long> exception){
```

Comment:
is this useful? why the parameter is named as `exception`. should be something like `alertConfigIds`?

Reply:
Useful for debugging and generating reports quickly. Renamed it to `excludeIds` and included a javadoc.